tests/sort-compiler-output-test/Makefile

```
@@ -1,11 +1,8 @@
 test: clean errors1-sorted.actual
-	diff errors1-sorted.goal errors1-sorted.goal
+	diff errors1-sorted.goal errors1-sorted.actual
 
 errors1-sorted.actual: ../../sort-compiler-output
 	../../sort-compiler-output < errors1.txt > $@
 
-# errors1-sorted.actual: ../../sort-compiler-output
-# 	../../sort-compiler-output < errors1.txt > $@
-
 clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)